src/haz3lcore/tiles/Whitespace.re

```diff
@@ -2,15 +2,45 @@ open Sexplib.Std;
 [@deriving (show({with_path: false}), sexp, yojson)]
```

Comment:
calling this Whitespace still seems confusing -- in the literature, this is known as secondary notation, so maybe we should rename this Secondary